### ISSUE TITLE
Snowflake comments with 0.20-rc2 create a query per comment

### ISSUE THREAD

**Issue by joshuataylor** [dbt-labs/dbt#3541]
### Describe the bug
With 0.19, having comments creates a query like this after the table is built:
```
alter table foo.bar.baz alter   
        some_table COMMENT $$My super awesome comment$$ ,
        something_else COMMENT $$Hello there$$ ,
```
Now it does a query per comment, which drastically reduces the speed of building large tables:
```
comment if exists on column foo.bar.baz.some_table is $$My super awesome comment$$;
```

This is because of https://github.com/dbt-labs/dbt/pull/3149 , I believe this a good fix but can lead to long build times for CI environments.

### Steps To Reproduce
- Install 0.20.0-rc2
- Add a table with comments with Snowflake
- Have multiple queries

### Expected behavior
A single alter table.

### System information
**Which database are you using dbt with?**
- [ ] postgres
- [ ] redshift
- [ ] bigquery
- [x] snowflake
- [ ] other (specify: ____________)

**The output of `dbt --version`:**
```
installed version: 0.20.0-rc2
   latest version: 0.19.2

Your version of dbt is ahead of the latest release!

Plugins:
  - snowflake: 0.20.0rc2
```

**The output of `python --version`:**
Python 3.8.6

**Comment by joshuataylor** [comment]
This is not a bug.

**Comment by jtcohen6** [comment]
@joshuataylor You're right, the fix in #3149 will slow down column-level `persist_docs` for tables with many many columns. My sense at the time is that it's worth trading the reduction in speed for the reduction in errors.

Based on what you're seeing anecdotally, how much slower is drastically slower? We could think about reworking this again for v0.21, by checking column metadata, verifying that the user-specified columns exist, and then running a single `alter` statement.

**Comment by joshuataylor** [comment]
This is awesome, thanks so much, will try it out. It was taking about 2 extra minutes, which is painful when you are charged for the larger warehouse sizes with SF. :-)
